<launch>
    <arg name="config" value="$(find apc_vision)/config.yaml"/>

    <include file="$(find softkinetic_launch)/launch/creative_senz3d.launch">
        <arg name="camera_name" value="camera_left"/>
        <!--<arg name="device_index" value=""/>-->
        <arg name="device_serial_number" value="K8VF0780347000071P"/>
        <arg name="confidence_threshold" value="100"/>
        <arg name="rgb_resolution" value="WXGA_H"/>
    </include>

    <node name="left_palm_camera_tf" pkg="tf" type="static_transform_publisher"
          args="0.005 0.06935 -0.138 0.191986218 3.28121899 -0.13962634 /arm_left_link_7_t /camera_left_depth_optical_frame 100"/>

    <node name="sample_server" pkg="apc_vision" type="sample_server_node" output="screen" cwd="node"/>
    <node name="process_server" pkg="apc_vision" type="process_server_node" output="screen" cwd="node" args="$(arg config)"/>
</launch>
